src/main/java/seedu/address/MainApp.java

```diff
@@ -32,9 +28,15 @@
 import seedu.address.storage.Storage;
 import seedu.address.storage.StorageManager;
 import seedu.address.storage.UserPrefsStorage;
+>>>>>>> 9be1c895c54ada063db055dc4b099ac35b753363
```

Comment:
This line is added by Git to show you where are the conflicting portions between your file and the team repo's file. You should resolve the conflict and remove this line before committing the file again. Please refer to the textbook portion below. https://nus-cs2103-ay1819s2.github.io/cs2103-website/se-book-adapted/chapters/gitAndGithub.html#merge-conflicts